File: Problem Set 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)

{
    int Height, Row, Free, Colum;

    // Checking for an input number
    do
    {
        Height = get_int("Height:");
    }
    while (Height < 1 || Height > 8);

    // print row
    for (Row = 0; Row < Height; Row++)
    {
        for (Free = Height - 1 ; Free > Row;  Free--)
        {
            printf(" ");
        }

        // print colume
        for (Colum = 0; Colum <= Row; Colum++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Problem Set 6/sentimental-mario-less/mario.py
# TODO

while True:
    try:
        hight = int(input('Hight: '))
        if hight > 0 and hight <= 8:
            break
    except:
        False

for row in range(hight):
    for colume in range(hight):
        if (row + colume < hight - 1):
            print(' ', end='')
        else:
            print('#', end='')
    print()